lib/yext/api/utils/configuration.rb

```diff
@@ -55,6 +55,23 @@ module Utils
       #
       # sandbox
       #   Boolean that indicates if the gem should use the production or sandbox URL.
+      #
+      # default_callback_processor
+      #   The default class to be used to process webhook messages that are called.
+      #   The code will warn about any missing webhook messages in a default processor.
+      # set_callback_processor(callback_function_name, processor)
+      #   The class to be used to process webhook messages for a specific callback.
+      #   If the method callback_function_name is not a public instance method for processor, an error
+      #   will be raised.
```

Comment:
Not sure where else to put this, but faraday supports default retries. We should check if that is enabled, and setup if it isn't https://github.com/lostisland/faraday/blob/master/lib/faraday/request/retry.rb#L11